tests/run-tests.sh

```diff
@@ -3,18 +3,50 @@
 set -euvo pipefail
 
 # This script assumes that it is running from the root of the Wit workspace.
-
 api_scala_sifive_path=./api-scala-sifive
+tests_path=$api_scala_sifive_path/tests
+coursier=./scala/coursier
+
+# Install jq if it is not already (as in environment-blockci-sifive)
+if hash jq 2>/dev/null; then
+  # jq already installed
+  jq=$(which jq)
+else
+  jq_linux_url=https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
+  jq=./jq
+  curl -L -o $jq $jq_linux_url
+  chmod +x $jq
+fi
+
 
 wake --init .
 
+
 # This is gross because we don't have a way of preventing Wake from
 # automatically picking up .wake files that only make sense in specific contexts
```

Comment:
It's not in master yet, but I've been look at putting ignorefile support into wake, if that happens to help you remove some `ln`
https://github.com/sifive/wake/pull/359

Reply:
Yeah that would clean this up